README.md

```diff
@@ -20,7 +20,7 @@ First create a conda environment and install relevant dependencies:
 conda create -y -n rescript
 conda activate rescript
 conda install \
-  -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.5/tested/ -c defaults \
+  -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.7/tested/ -c defaults \
```

Comment:
you install 2023.7 here, but then mention 2023.9 below. Why not just use the latest?

Reply:
I wanted to highlight two ways to install RESCRIPt, as many of the recent versions can be installed as we've been doing. But not so with 2023.9... 

I also figured, we'd not merge this PR until the next release anyway. This will serves as our scratch space. :-)

Reply:
okay good point, and I like the idea of keeping install instructions for pre-2023.9, but maybe we should change the header line accordingly